### DISCUSSION TITLE
Welcome to PS4-PKG-Tool Discussions!

### DISCUSSION THREAD

**Discussion by pearlxcore** [pearlxcore/PS4PKGTool#17] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sikanderfoark** [comment]
brother Kindly guide me how to use PS4 pkg tool 1.4

**Reply by pearlxcore** [reply]
Just add folder containing ps4 pkg. The program will load and display the pkg. You can set the program to scan/load pkg in subfolder by ticking 'recursive scan' checkbox.

**Comment by nikeymikey** [comment]
Hey, great tool, really useful. 

I have found that the program doesn't like Homebrew pkg's such as Apollo Save Manager or Payload Guest.  Scanning stops when one of these files is found. Not a biggie to me as I now keep these in a separate folder. 

I have also had random crashes on the Trophy tab.

Would it be possible to have a function to highlight any dupes that may be in the pkg list?

Thanks and keep up the great work :)

**Reply by nikeymikey** [reply]
1 more thing i just found....

When renaming all and using the [_VERSION_] string, any DLC will have a [ ] in the name as they do not seem to actually have a version number.  :)
